api_signaling.go

```diff
@@ -784,6 +789,11 @@ func (e *EventServerMessageSessionEntry) Clone() *EventServerMessageSessionEntry
 	}
 }
 
+type EventServerMessageSwitchTo struct {
+	RoomId  string          `json:"roomid"`
+	Details json.RawMessage `json:"properties,omitempty"`
```

Comment:
[According to documentation the message uses `details`](https://github.com/strukturag/nextcloud-spreed-signaling/pull/409/files#diff-9752dbbe894845f1e7cc6dfe36a6775c2b0c6b5633f9700c31d413bc44914ceaR1084) instead of `properties`.

Reply:
Good catch, changed. Thanks!